airflow/contrib/operators/bigquery_operator.py

```diff
@@ -383,7 +383,7 @@ class BigQueryCreateExternalTableOperator(BaseOperator):
     :type schema_fields: list
     :param schema_object: If set, a GCS object path pointing to a .json file that
         contains the schema for the table. (templated)
-    :param schema_object: string
+    :type schema_object: string
```

Comment:
Please convert this to str as well :)

Reply:
I am creating a follow PR to do this as this (`string` -> `str`) is going to affect 100s of files.

Reply:
Awesome!